cf-stack.json

```diff
@@ -43,16 +43,7 @@
             {
               "Effect": "Allow",
               "Action": [
-                "cloudformation:CreateStack",
-                "cloudformation:CreateUploadBucket",
-                "cloudformation:DeleteStack",
-                "cloudformation:DescribeStackEvents",
-                "cloudformation:DescribeStackResource",
-                "cloudformation:DescribeStackResources",
-                "cloudformation:ListStackResources",
-                "cloudformation:UpdateStack",
-                "cloudformation:DescribeStacks",
-                "cloudformation:SetStackPolicy"
+                "cloudformation:*"
```

Comment:
Bylo potreba rozsirit s updatem na Serverless 3 - stejne je to i u Developer portalu jsem koukal:
https://github.com/keboola/developer-portal/blob/main/cf-resources.json#L188